rfcs/text/0000-matrix-service.md

```diff
@@ -0,0 +1,406 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Matrix Service
+
+## Summary
+
+The existing Matrix Service is a prototype of a DCP product that is capable of enabling third party data consumers to perform integrative analysis across HCA experiments. In its current implementation, this is achieved by 1) delivering a RESTful API that enables data consumers to generate expression matrices defined by a metadata query against an index of cells, and 2) an index of cells in the HCA that belong to projects for which a secondary analysis pipeline is available and complete. This RFC details and requests community review on the prototype's major design decisions surrounding i) the Matrix Service API and ii) the service's cell-centric data model.
+
+## Authors
+
+[Calvin Nhieu](cnhieu@chanzuckerberg.com)
+[Marcus Kinsella](mkinsella@chanzuckerberg.com)
+
+## Shepherd
+ [Brian Raymor](braymor@chanzuckerberg.com)
+
+## Motivation
+
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests, processes, and stores single-cell data, making that data easily available to a wide audience of researchers. The single-cell data type of greatest interest to most researchers is the expression matrix, the cell-by-gene expression values that are the starting point for many downstream analyses. To support this most commonly requested data type, the DCP offers a specialized interface, the DCP matrix service, which enables investigators to easily access and perform integrative analysis across HCA expression data.
+
+### User Stories
+
+As a data consumer, I would like to access gene expression data from the DCP so that I can perform downstream integrative analysis across HCA experiments.
+
+- I would like to be able to receive an expression matrix comprised of cells that satisfy a set of metadata conditions.
+- I would like to receive metadata providing the experimental and scientific context about the cells in my matrix.
+- I would like to receive my matrix in a format that is compatible with standard single-cell analysis tools.
+
+## Scientific "guardrails"
```

Comment:
Consider adding guidance related to multiple species:


[**August 6 - Guidance from AWG**](https://docs.google.com/document/d/1YcSg2DhePi1Ozb0ty7_pGdAPiPeN9B6WaDhuN0PiYRU/edit#heading=h.qf4cth7gazsu)

Consider a "give me all kidney cells" query: 
- Only return human kidney cells, mouse requires an explicit "from mouse" in the query
  - This assumes all queries are human by default.
  - This is the requested default for a query not specifying species.